deny.toml

```diff
@@ -29,6 +29,7 @@ confidence-threshold = 0.8
 exceptions = [
     { allow = ["Zlib"], name = "adler32", version = "*" },
     { allow = ["ISC", "MIT", "OpenSSL"], name = "ring", version = "*" },
+    { allow = ["Unicode-DFS-2016"], name = "unicode-ident", version = "*"},
```

Comment:
```suggestion
    # The Unicode-DFS--2016 license is necessary for unicode-ident because they
    # use data from the unicode tables to generate the tables which are
    # included in the application. We do not distribute those data files so
    # this is not a problem for us. See https://github.com/dtolnay/unicode-ident/pull/9/files
    { allow = ["Unicode-DFS-2016"], name = "unicode-ident", version = "*" },
```